packages/account/src/Components/poi/idv-document-submit/utils.js

```diff
@@ -189,3 +195,11 @@ const idv_document_data = {
         },
     },
 };
+
+//skip validation list for testing smileidentity
+export const idv_test_document_whitelist = new Map([
+    ['gh', { drivers_license: 'B0000000', passport: 'G0000000', ssnit: 'CO00000000000', voter_id: '0000000000' }],
```

Comment:
`CO00000000000`

Just wanted to check, but is it intended that there is an O in the not a 0 in the beginning?

Reply:
thanks for attentiveness, will check it with doc